CONTRIBUTING.rst

```diff
@@ -513,7 +513,7 @@ Now you can run your tests:
 
 ::
 
-   python -m nox -e "pytest-3.7(coverage=False)" -- tests/unit/cli/test_batch.py
+    python -m nox -e "test-3(coverage=False)" -- tests/unit/cli/test_batch.py
```

Comment:
Can you remove the extra space here on this line? It might mess up the formatting.